### PR TITLE
Translate "Content Slider" and "Slideshow" in Beaver Builder

### PR DESCRIPTION
**Pull request by strategio**
This is based on https://github.com/OnTheGoSystems/wpml-page-builders-beaver-builder/pull/10.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlpb-163

### CODE REVIEW

File: src/media/modules/class-wpml-beaver-builder-media-node.php [hunk 1]
```diff
@@ -9,5 +9,5 @@ public function __construct( WPML_Page_Builders_Media_Translate $media_translate
 		$this->media_translate = $media_translate;
 	}
 
-	abstract function translate( $node_data, $source_lang, $target_lang );
+	abstract function translate( $node_data, $target_lang, $source_lang );
```
Comment (strategio) [inline src/media/modules/class-wpml-beaver-builder-media-node.php:12]: Fixing the arguments order.

File: src/media/modules/class-wpml-beaver-builder-media-node-content-slider.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+class WPML_Beaver_Builder_Media_Node_Content_Slider extends WPML_Beaver_Builder_Media_Node {
+
+	private $property_prefixes = array(
+		'bg_',
```
Comment (brucepearson) [inline src/media/modules/class-wpml-beaver-builder-media-node-content-slider.php]: @strategio Maybe add a comment after each of these so it's clearer to see what properties you are handling.

**Review by brucepearson — APPROVED** [review]

**Comment by strategio** [comment]
@OnTheGoSystems/wpml, review please :)

**Review by adrianosferreira — APPROVED** [review]
+1

**Pull request merged**